scripts/delayed.js

```diff
@@ -1,7 +1,32 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './aem.js';
+import { sampleRUM, loadScript } from './aem.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/*
+  * Returns the environment type based on the hostname.
+*/
+function getEnvType(hostname = window.location.hostname) {
+  const fqdnToEnvType = {
```

Comment:
Can we move this to a common utility?  api/user.js does something really similar and it seems we might want to centralize this kind of env-aware logic.

Reply:
Moved it to util.js.